docs/content/tutorials/risky-containers/_index.md

```diff
@@ -0,0 +1,301 @@
+|-----|-----------------------------------|---------------|
+| 1   | drasidemo.azurecr.io/my-app:0.1   | Security Risk |
+| 2   | docker.io/library/redis:6.2.3-alpine   | Compliance Issue  |
+
+
+### Deploy Pods
+
+The following command will deploy two Pods of `my-app`, one with version `0.1` and one with version `0.2`. 
+
+```shell
+kubectl apply -f ./resources/my-app.yaml  
+```
+
+
+### Store Kubernetes credentials in a secret
+
+Before we can create a Kubernetes source, we need the credentials of the cluster that the source will connect to. The way to get these credentials will differ depending on how you are running Kubernetes.  The scripts below will extract the credentials of your current Kubernetes context and store them in a secret, to be referenced by the Drasi Kubernetes source.
+
+> If you are running using Github Codespaces or the VS code dev container, then use the `k3d` script.
+
+{{< tabpane >}}
+
+{{% tab header="k3d" text=true %}}
+
+```shell
+k3d kubeconfig get k3s-default | sed 's/0.0.0.0.*/kubernetes.default.svc/g' | kubectl create secret generic k8s-context --from-file=context=/dev/stdin -n drasi-system
+```
+
+{{% /tab %}}
+
+{{% tab header="Kind" text=true %}}
+
+```shell
+kind get kubeconfig | sed 's/127.0.0.1.*/kubernetes.default.svc/g' | kubectl create secret generic k8s-context --from-file=context=/dev/stdin -n drasi-system
+```
+
+{{% /tab %}}
+
+{{% tab header="AKS" text=true %}}
+
+```shell
+az aks get-credentials --resource-group <resource-group> --name <cluster-name> --file - | kubectl create secret generic k8s-context --from-file=context=/dev/stdin -n drasi-system
+```
+
+{{% /tab %}}
+
+{{% /tabpane %}}
+
+
+### Deploy the sources
+
+The following command will deploy the PostgreSQL and the Kubernetes sources.
+
+
+```shell
+drasi apply -f ./resources/sources.yaml
+```
+
+The following command will wait for the source to be ready.
+
+```shell
+drasi wait -f ./resources/sources.yaml
+```
+
+
+### Deploy Continuous Query
+
+Next, we will create a Continuous Query that will join the rows in the `RiskyImage` table to Pods that are running inside the Kubernetes cluster.  The Kubernetes source will create graph nodes that match the Kubernetes API.  The container information that we are interested in is nested in this object in the `status.containerStatuses` array.  We are looking to use the `image` property on the entries in this array to join to the `Image` column of the `RiskyImages` table.
+
+Here is a sample of the Pod payload from the Kubernetes API.
+```json
+{
+    "apiVersion": "v1",
+    "kind": "Pod",
+    "metadata": {    
+        "creationTimestamp": "2024-12-18T20:00:13Z",
+        "name": "my-app-1"
+    },
+    "spec": ...,
+    "status": {
+        "conditions": [
+          ...
+        ],
+        "containerStatuses": [
+            {
+                "containerID": "containerd://...",
+                "image": "drasidemo.azurecr.io/my-app:0.1",
```

Comment:
I am wondering if we should upload the `my-app` images to our ghcr so that we can remove this dependency on ACR